README.md

```diff
@@ -83,6 +83,12 @@ XCTMain([
     * [x] Deleting
     * [x] Fetching by Coupon ID
     * [x] Listing All Coupons (With filters)
+* [x] Plans
+    * [x] Creating
+    * [x] Updating
+    * [x] Deleting
+    * [x] Fetching by Plan ID
+    * [x] Listing All Plans (With filters)
 * [ ] Disputes
```

Comment:
Just cause this list is now getting so big, could you move `[ ] Disputes` above `[ ] Cards` just so all the completed items are on top and uncompleted at the bottom.